CHANGELOG.md

```diff
@@ -36,9 +35,11 @@ and changes prior to that are (unfortunately) done retrospectively. Critical ite
  - Fix for empty docker namespaces in private repositories
  - Fix Docker environment parsing
  - Revert lolcow easter egg
-
-
-
+ - Fix "Duplicate bootstrap definition key" triggered by comments and blank spaces
+ - Fix for docker permission error when downloading multiple layers
+ - Fix parsing of docker tags and version
```

Comment:
registry (including port), namespace, tags, and version (whew it's a lot to parse!)